### PR TITLE
01 seed and serve api 

### PR DESCRIPTION
**Pull request by cjm771**
[WIP] - I'll change to mergeable when I have a working api. Please see roadmap below

 - [x] Parsed data and downloaded static files
 - [x] Uploaded files to s3
 - [x] autogenerating sql seed file
 - [x] populated sql database
 - [x] API working for routes outlines in README
 - [x] rough parsing method for gpx files from S3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "server-dev": "nodemon --inspect server/index.js",
-    "seed-database": "psql 9trails-paths < ../db/schema.sql"
+    "seed-database": "psql 9trails-paths < db/schema.sql"
```
Comment (McCloudFace) [inline package.json]: Soooo, I take it the `[Dont' Merge]` means just that? On line 12 you may want to perform a check of the fetzer valve before you engage the hyperdrive. Your bearings housing will also be more efficient during liftoff. Just a thought tho.

**Review by McCloudFace — COMMENTED** [review]
Sooooo, I'm guessing "[Don't Merge]" means just that? If so, this looks super cool - definitely want to try out your Google Sheets wizardry! Nice!

**Comment by cjm771** [comment]
Yeah [dont merge] means in progress (as per Leslie), After I clear out all these checkboxes it'll be ready to go but a pull request is nice for my wonderful reviewers to know whats coming / sneak peak  and can drop on the boards.

**Comment by McCloudFace** [comment]
Love the checkboxes tip and the progress bar - very cool.

**Comment by cjm771** [comment]
Alright updated with working api, see readme for specifics! Merge / review at discretion.

**Comment by McCloudFace** [comment]
This is really well done @cjm771 ! Nice work!

**Comment by McCloudFace** [comment]
Wasn't sure if you wanted the `01_seed_and_serve_api` branch deleted, so left it here for you to decide. Nice work.

**Comment by cjm771** [comment]
Ah thanks! Ok to delete , ill do so!

File: db/db.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      await Promise.all(willResolveFormatting).then((resolvedVals) => {
+        resolvedVals.forEach((resolvedVal, i) => {
+          formattedObject[formatKeys[i]] = resolvedVal;
+        });
+  
+      });
+      // return object
+      return formattedObject;
+    } catch (e) {
+      throw e;
```
Comment (LesliePajuelo) [inline db/db.js:70]: Normally you throw an error to a catch. Where is this throw going? (hint: killing your program)

Also, FYI, this format will only ever catch the last failed promise. Which might be intentional that you don't want a catch for every failed promise.

**Pull request merged**